assets/javascript/app.js

```diff
@@ -1,13 +1,12 @@
 $(document).ready(function() {
     var beer = [];
     var location;
-    var radius = 10;
+    var distance = 10; // nearby brewery distance
```

Comment:
Changed `radius` global var to `distance` as it was more representative of what we are attempting to achieve/pass in.